projects/js-packages/publicize-components/src/components/connect-button/index.js

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@automattic/jetpack-components';
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { requestExternalAccess } from '../../utils/request-external-access.js';
+
+const ConnectButton = ( { connectUrl, onClose, className } ) => {
+	const requestConnection = useCallback(
+		() => requestExternalAccess( connectUrl, onClose ),
+		[ connectUrl, onClose ]
+	);
+	return (
+		<Button className={ className } variant="secondary" onClick={ requestConnection }>
+			{ __( 'Connect', 'jetpack' ) }
```

Comment:
Instead of this, we probably want to pass in a child component or something that would be the service name and icon.

Reply:
Since this is in the same row as the service name and icon on the services list, this is fine.